src/mcts/search.cc

```diff
@@ -910,7 +929,18 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
           continue;
         }
       }
-      const float Q = child.GetQ(fpu, params_.GetLogitQ());
+      float Q = child.GetQ(fpu, params_.GetLogitQ());
+      if (use_moves_left && moves_left != 0) {
+          const float max_moves_scale = params_.GetMovesLeftScale();
+          const float M = std::max(std::min(child.GetM(best_node_m) - best_node_m, max_moves_scale), -max_moves_scale) / max_moves_scale;
```

Comment:
I didn't put this in a function because I don't think `Trim` or any other short name really describes it. It's short enough and isn't used anywhere else. I added a comment above it to describe what it does.